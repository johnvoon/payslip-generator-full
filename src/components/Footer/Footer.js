import React from "react";

// eslint-disable-next-line no-unused-vars
const Footer = props => {
  return (
    <footer className="footer text-center">
      <ul className="list-inline">
        <li>Payslip Generator 2017</li>
      </ul>
    </footer>
  );
};

export default Footer;
