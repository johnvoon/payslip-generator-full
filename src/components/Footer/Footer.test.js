import React from "react";
import Footer from "./Footer";
import { shallow } from "enzyme";

describe("Footer", () => {
  const renderedComponent = shallow(<Footer />);

  it("should match snapshot", () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
