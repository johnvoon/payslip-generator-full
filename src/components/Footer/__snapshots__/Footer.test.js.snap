exports[`Footer should match snapshot 1`] = `
<footer
  className="footer text-center">
  <ul
    className="list-inline">
    <li>
      Payslip Generator 2017
    </li>
  </ul>
</footer>
`;
